plugins/Login/templates/invitation.twig

```diff
@@ -1,48 +1,55 @@
+{% set isCenteredLogo = false %}
+{% set showImageDiv = true %}
+{% set imageToShow = 'plugins/Login/images/invitation-image.png' %}
+{% set colClassOverride = 'col l4 offset-l2 m12' %}
+{% set bodyClass = 'onBoarding' %}
 {% extends '@Login/loginLayout.twig' %}
 
 {% block loginContent %}
-    <div class="contentForm invitationForm">
-        <div class="card">
-            <div class="card-content">
-                <div class="card-title">
-                    {{ "Login_InvitationTitle"|translate }}
-                </div>
+    <div class="contentForm invitationForm" id="invitationForm">
+        <p class="uppercase header-p m-top">{{ 'General_CreatePassword'|translate }}</p>
+        <p class="header-p">{{ 'Login_CreatePasswordDescription'|translate }}</p>
 
-                {% if AccessErrorString is defined %}
-                    <div vue-entry="CoreHome.Notification"
-                         noclear="true"
-                         context="error">
-                        <strong>{{ 'General_Error'|translate }}</strong>: {{ AccessErrorString|raw }}<br/>
+        {% if AccessErrorString is defined %}
+            <div vue-entry="CoreHome.Notification"
+                 noclear="true"
+                 context="error">
+                <strong>{{ 'General_Error'|translate }}</strong>: {{ AccessErrorString|raw }}<br/>
+            </div>
+        {% endif %}
+        {% block content %}
+            <form method="post"
+                  action="{{ linkTo({'module': loginPlugin, 'action': 'acceptInvitation', 'token': null}) }}">
+                <input type="hidden" name="token" value="{{ token }}"/>
+                <div class="row">
+                    <div class="col l10 s12 input-field">
+                        <input type="text" name="email" value="{{ user.email }}" size="20" readonly="readonly"
+                               tabindex="0"/>
+                        <label>{{ 'UsersManager_Email'|translate }}</label>
```

Comment:
```suggestion
                        <label for="email">{{ 'UsersManager_Email'|translate }}</label>
```

Reply:
@michalkleiner This field is non-editable and adding `for` won't help much as clicking on the label won't focus the input

Reply:
@michalkleiner updated as suggested